src/Microsoft.Health.Dicom.Api/Features/Exceptions/ApiVersionNotSupportedException.cs

```diff
@@ -0,0 +1,17 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Globalization;
+using Microsoft.Health.Dicom.Core.Exceptions;
+
+namespace Microsoft.Health.Dicom.Api.Features.Exceptions;
+
+public class ApiVersionNotSupportedException : BadRequestException
```

Comment:
Is it sufficient to instead create a `BadRequestException` instead of creating a new exception type?

Reply:
I prefer strong tying to consolidate the messaging for that error